src/inoreaderapi.cpp

```diff
@@ -202,10 +201,8 @@ bool InoreaderApi::mark_all_read(const std::string& feedurl)
 		feedurl.substr(strlen(INOREADER_FEED_PREFIX));
 	std::vector<std::string> elems = utils::tokenize(real_feedurl, "?");
 	real_feedurl = utils::unescape_url(elems[0]);
-	std::string token = get_new_token();
 
-	std::string postcontent =
-		strprintf::fmt("s=%s&T=%s", real_feedurl, token);
+	std::string postcontent = strprintf::fmt("s=%s&T=%s", real_feedurl);
```

Comment:
I believe the `&T=%s` should be removed here as well.

Reply:
Thanks! I force-pushed an updated commit.